typescript/cli/src/deploy/warp.ts

```diff
@@ -652,6 +652,11 @@ async function updateExistingWarpRoute(
 
   await promiseObjAll(
     objMap(expandedWarpDeployConfig, async (chain, config) => {
+      if (multiProvider.getProtocol(chain) !== ProtocolType.Ethereum) {
```

Comment:
Optional Nit: Probably personal preference, but I'd filter them out from `expandedWarpDeployConfig`

Reply:
thought i'd keep the condition in the loop so we can always adjust if we support more protocol types here